src/lib/components/NglMoleculeViewer.react.js

```diff
@@ -83,14 +83,21 @@ NglMoleculeViewer.propTypes = {
     id: PropTypes.string,
 
     /**
-     * The height and the width (in px) of the container
+     * Dash-assigned callback that should be called whenever properties change.
+     */
+    setProps: PropTypes.func,
+
+    /**
+     * The width (in px or as a number) of the container
      * in which the molecules will be displayed.
-     * It should be in JSON format.
      */
-    viewportStyle: PropTypes.exact({
-        width: PropTypes.string,
-        height: PropTypes.string,
-    }),
+    viewportWidth: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+    /**
+     * The height (in px or as a number) of the container
+     * in which the molecules will be displayed.
+     */
+    viewportHeight: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Can we just use `width` and `height`? That matches what we do for the overall size of a lot of other components, and doesn't seem ambiguous to me so I don't think we need the `viewport` qualifier.

Reply:
Updated in c39b8a6.